lms/templates/instructor/instructor_dashboard_2/cohorts.underscore

```diff
@@ -24,6 +24,9 @@
     </a>
 </div>
 
+<!-- Add modal -->
+<div class="cohort-management-add-modal"></div>
```

Comment:
NIT: I'd sway away from using the term modal in class names and in describing this element. From a UX/UI PoV, modal implies taking focus and constraining it to a particular DOM element (hence the usual overlay + save/cancel actions). Any chance we could use a term more suitable to this case, like ``cohort-management-add-form`` or ``cohort-management-add``?